treebeard/tests/test_treebeard.py

```diff
@@ -2356,12 +2355,12 @@ def test_treebeard_js(self):
         rendered = template.render(context)
         expected = ('<script type="text/javascript" src="jsi18n"></script>'
                     '<script type="text/javascript" '
-                    'src="/treebeard/treebeard-admin.js"></script>'
-                    '<script>(function($){'
+                    'src="' + static('treebeard/treebeard-admin.js') + '"></script>'
+                    '<script>(function($){{'
                     'jQuery = $.noConflict(true);'
-                    '})(django.jQuery);</script>'
+                    '}})(django.jQuery);</script>'
```

Comment:
These two lines have too many curly braces now, it seems.

Reply:
Removed!